.devcontainer/devcontainer.json

```
@@ -0,0 +1,15 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "customizations": {
```

More improvements: by default, it opens the README, but doesn't do so immediately. So if you start a Codespace and are looking at some file, it will abruptly open README.md.

I believe you can set [`openFiles`](https://containers.dev/supporting#github-codespaces) like this:
```suggestion
  "customizations": {
		"codespaces": {
		  "openFiles": []
		},
```

Empty array would be ideal, but if it doesn't work then you can open `playgrounds/vite/src/App.tsx`.